File: day1/part1.go
package main

import (
	"fmt"
	"log"
	"os"
)

func main() {
	f, err := os.ReadFile("./input.txt")
	if err != nil {
		log.Fatalf("input file not found")
	}

	data := string(f)
	level := 0
	for _, val := range data {
		if val == '(' {
			level += 1
		} else if val == ')' {
			level -= 1
		}
	}

	fmt.Println(level)

}

File: day1/part1.py
def main():
    with open("input.txt", "r") as f:
        p = f.readline()
    counter = 0
    for i in p:
        if i == '(':
            counter += 1
        elif i == ')':
            counter -= 1
    print(counter)


if __name__ == '__main__':
    main()
